README.md

```diff
@@ -8,6 +8,13 @@ This is an incubation space, and so is slightly anarchic: the idea is to "let ma
 
 Our expectation is that you are seeking collaboration and co-operation, and that you wish to facilitate other people to contribute to your project.
 
+To get a comprehensive understanding, please read [the policies section](#policies) below. Since sharing and collaboration are based on trust and benevolence, here are some important key points of the impact having a project in this space:
+
+* _fsprojects_ is not a legal organization, and it is not part of the [F# Software Foundation](https://fsharp.org), although well-known FSSF members are part of it by hosting or contribution.
+* _fsprojects_ is currently maintained by Sergey Tihon and Don Syme.
+* "Transfering" your project to _fsprojects_ does **not** mean that you give away ownership or any rights apart from the ones you grant in your project's license.
+* Moving your project out of _fsprojects_ is always possible.
```

Comment:
This one as well https://github.com/fsprojects/fsprojects-admin/blob/master/README.md?plain=1#L39